### ISSUE TITLE
Error when drawing multiple bands per row

### ISSUE THREAD

**Issue by YutingHan317** [gforge/forestplot#69]
Hi, I am trying to draw multiple bands per row. And I have made the label texts the same across the two groups. For the database, please take a look at the attachments. To draw the plot, I used my old codes (see below), which were written 1-2 years ago and were completely functional. However, an error appeared this time. 
[Subtype-male-model1-1th.csv](https://github.com/user-attachments/files/17537072/Subtype-male-model1-1th.csv)

>Error in forestplot.grouped_df(., labeltext = c(V1, V5, V6), mean = V2,  : 
  There are seem be invalid the labels: Continuous > 2.27 (1.72-3.00) > 1.48 (1.23-1.77), Continuous > 2.27 (1.72-3.00) > 1.48 (1.23-1.77)
 appear in the wrong position.

The codes are as follows.
```R
library(dplyr)
library(forestplot)
library(ggplot2)
library(magrittr)
library(data.table)

sub_male_m1_1th <- fread("Subtype-male-model1-1th.csv")

colnames(sub_male_m1_1th) <- paste0("V", seq_len(ncol(sub_male_m1_1th)))

sub_male_m1_1th %>%
group_by(V8) %>% 
  forestplot(labeltext = c(V1, V5, V6),
             mean = V2, lower = V3, upper = V4)
```

R-version: 4.3.2 and the Package version: 3.1.3&3.1.5（I've tried both）.

And when I removed the rows with NA, the codes worked. But I need those rows.

Thank you for any help you can give me!

Best wishes,

Yuting

**Comment by gforge** [comment]
This is a tricky one. There are a few issues (apart from the bad language in the error message):

- You have headers appearing twice making it non-unique (see categorical)
- The categories are not equal between the groups
- Multiple labels V1, V5, V6

I've been looking at fixing this with some form of adding empty rows for the group but I'm close to giving up. This will probably never scale properly and I think I will need to restrict the code more.

**Comment by gforge** [comment]
I fixed everythin in 3.1.6 except for the non-unique labels. I think this use case is probably not that suited for dply-syntax

**Comment by YutingHan317** [comment]
Dear Max,

Thank you for your assistance. I appreciate it and apologize for the delayed response. I must have missed the reminder.

Kind regards,
Yuting
